packages/wallets/src/components/WalletCard/WalletCard.tsx

```diff
@@ -84,7 +84,7 @@ const WalletCard: React.FC<TProps> = ({
                     </div>
                 </WalletGradientBackground>
             </div>
-        </button>
```

Comment:
This `onClick` behaviour is only added for responsive touch events, other than that the card should be a `div` container for desktop screens

Reply:
You can add `role='button' ` to your div. The sonarcloud error will go away aswell (I think).

Reply:
Nahh it doesnt, it still complains, u cant put `role` on a `div`

Reply:
@adrienne-deriv one thing you can do if you want to keep it a button but not cause the background issue is to add a variant i.e `wallets-card__interactive` and put the background/border: none inside that variant block
```
.wallets-card {
     &__interactive {
          background: none;
          border: none;
     }
}
```
and add this variant only on the components which needs it

Reply:
but the thing is this is not a button, its just a `div` container 😅 
The `onClick` was only added for touch events in responsive
I think adding the `.wallets-card__interactive` styling would be more confusing later on